src/Input/Input.jsx

```diff
@@ -69,10 +70,11 @@ export const Input = forwardRef(
     const currentIcRight = useMemo(() => {
       if (loading) return 'loading'
       else if (error || warning) return 'alertCircle'
+      else if (inputType === 'search') return value ? 'close' : 'magnify'
```

Comment:
What if the user wants to set the `icRightIcon`?

Reply:
8b06ce4738885b4dfa24583b52dae9b3cd80df84